### ISSUE TITLE
The daily update of the website "Lensfun's coverage" stopped on March 19th.

### ISSUE THREAD

**Issue by tuxfanx** [lensfun/lensfun#1743]
[Lensfun's coverage](https://wilson.bronger.org/lensfun_coverage.html) stopped on March 19th. with 1199 counts. New entries in the database are no longer listed.

**Comment by bronger** [comment]
I had trouble with the server which should have been resolved shortly before this issue. Currently, my ownCloud sync aborts because I cannot delete upload directory a87fe4_ntsarb. Is there anything special about it?

**Comment by tuxfanx** [comment]
I worked on a87fe4_ntsarb. Normally I always copy the directory. Unfortunately I had forgotten that in this case. There were still sidecar files from me in it. I have now tried to delete them. Unfortunately, one xmp file is defective and cannot be deleted. That's probably the cause. The images in the directory are no longer required and can be deleted.
I'm sorry my mistake is the cause of the problem. At the moment I don't know how to remove the defective xmp file.
